pt-BR/mail/chrome/messenger/addressbook/abCardOverlay.dtd

```diff
@@ -85,7 +85,7 @@
 <!ENTITY allowRemoteContent.label       "Allow remote images in HTML mail.">
 <!ENTITY allowRemoteContent.accesskey   "g">
 
-<!ENTITY Phones.box                     "Phones">
+<!ENTITY Phones.box                     "Telefones">
 <!ENTITY WorkPhone.label                "Comercial:">
```

Comment:
CORRECTIONS NEEDED instead of <!ENTITY WorkPhone.label                "Trabalho:">